RELEASE_NOTES.md

```diff
@@ -3,8 +3,33 @@ Release notes
 
 ### Next
 
-* Included support for compilation on Java 9 and 10. Compilation on Java 9 or 10 will produce a multi-release JAR that will run on Java 8, 9, and 10. This release was compiled on Java 8 and does not contain a multi-release JAR.
-* Changed metadata health check names to use the URI of the metadata they are trying to resolve
+#### Connect to Verify using only the VSP
+
+This release adds the ability to connect to GOV.UK Verify using only the Verify Service Provider (VSP). This means services can connect without needing to host a [Matching Service Adapter (MSA)](https://github.com/alphagov/verify-matching-service-adapter). This makes it easier to connect to Verify and reduces maintenance tasks once connected.
+
+When used alone, the VSP provides MSA functionality. It can:
```

Comment:
That might be a bit misleading.  The core job of the MSA was to talk to the RP's Local Matching Service, which the VSP will not be doing.

Reply:
I agree. Changed to say 'the required MSA functionality'.